### ISSUE TITLE
Meaning of eps in train.log

### ISSUE THREAD

**Issue by sathvikask0** [espnet/espnet#257]
What does eps mean in the train.log file. Is it learning rate or some smoothening factor?

**Comment by sw005320** [comment]
This is a tuning parameter (epsilon) in AdaDelta
https://docs.chainer.org/en/stable/reference/generated/chainer.optimizers.AdaDelta.html#chainer.optimizers.AdaDelta
https://pytorch.org/docs/stable/_modules/torch/optim/adadelta.html

We'll change the epsilon when we observe the degradation of the validation accuracy at some epochs. So, the log monitors this epsilon value.
